### ISSUE TITLE
couldn't find gem

### ISSUE THREAD

**Issue by coorasse** [javanthropus/rgl#1]
I can't write
gem 'rgl', :git => 'git://github.com/javanthropus/rgl.git'

like other gems. why?

**Comment by javanthropus** [comment]
There isn't a gemspec file in the root of the repository. You shouldn't need to add the gem that way though because RGL already available as a gem from rubygems.org.

-Jeremy

coorasse reply@reply.github.com wrote:

I can't write
gem 'rgl', :git => 'git://github.com/javanthropus/rgl.git'

like other gems. why?

---

Reply to this email directly or view it on GitHub:
https://github.com/javanthropus/rgl/issues/1

**Comment by coorasse** [comment]
ok. so if I want to fork I have to write my own gemspec and then point to my git repo, right?

**Comment by javanthropus** [comment]
Yes, that would be the process, but why do you want to install the gem this way in the first place? Is there a bug to fix or a feature you need?

-Jeremy

coorasse reply@reply.github.com wrote:

ok. so if I want to fork I have to write my own gemspec and then point to my git repo, right?

---

**Comment by coorasse** [comment]
it's just first time I run into this problem and I couldn't understand why. 
Now I learned something more about gems and better understood the mechanism.
I was looking for a complete library for graphs data structure but I can't find nothing with all the functionality I need:
I am looking for a library which include the possibility to define weighted graphs, and give some complex algorithms out of the box like community detecting and clustering.
I just found 'igraph' that is a C wrapper so I will try with it.
